webroot/results/admin/export_public.php

```diff
@@ -157,6 +154,10 @@ function exportPublic ( $sources ) {
         echo '.';  # shows both Apache and the user that the script is doing stuff and not hanging
       }
     }
+    //Check if any errors while fetching data
+    if (($error = mysql_error()) !== '') {
```

Comment:
So what happens if `mysql_error()` returns boolean false? The docs say that this returns an empty string, but my PsySH REPL disagrees. Either way, I feel like just checking empty string could introduce a bug unless my REPL is just being weird.

Docs:
> Returns the error text from the last MySQL function, or '' (empty string) if no error occurred.

REPL:
```
>>> mysql_error()
=> false
>>> mysql_error() === ''
=> false
>>> mysql_error() !== ''
=> true
>>> 
```